### PR TITLE
Sql job_points

### PR DESCRIPTION
**Pull request by some1here**
from windower

### CODE REVIEW

File: sql/job_points.sql [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+INSERT INTO `job_points` VALUES ('900', 'fly_high_effect', '5', '14');
+INSERT INTO `job_points` VALUES ('902', 'jump_effect', '3', '14');
+INSERT INTO `job_points` VALUES ('904', 'spirit_link_effect', '1', '14');
+INSERT INTO `job_points` VALUES ('906', 'wyvern_max_hp_bonus', '10', '14');
+INSERT INTO `job_points` VALUES ('908', 'dragon_breaker_duration', '10', '14');
+INSERT INTO `job_points` VALUES ('910', 'wyvern_breath_effect', '14', '1');
+INSERT INTO `job_points` VALUES ('912', 'high_jump_effect', '3', '14');
+INSERT INTO `job_points` VALUES ('914', 'wyvern_attr_Increase_effect', '10', '14');
+INSERT INTO `job_points` VALUES ('960', 'astral_flow_effect', '5', '15');
+INSERT INTO `job_points` VALUES ('962', 'avatar/spirit_accuracy_bonus', '1', '15');
```
Comment (xdemolish) [inline sql/job_points.sql]: remove all the avatar/ cause prolly gonna cause path issues

**Comment by xdemolish** [comment]
@teschnei

**Comment by bendangelo** [comment]
It looks good

**Comment by bendangelo** [comment]
I'm just going to merge this in. It's not even used.
Thanks for contributing :heart:

**Pull request merged**